File: c++/1101.cpp
#include <iostream>

using namespace std;

int main() {
    int x, y, soma, i, tmp;

    cin >> x >> y;
    while (x != 0 && y != 0 && x > 0 && y > 0) {
        soma = 0;
        if (y < x) {
            tmp = x;
            x = y;
            y = tmp;
        }
        for (i=x;i<=y;i++) {
            cout << i << " ";
            soma += i;
        }
        cout << "Sum=" << soma << "\n";
        cin >> x >> y;
    }

    return 0;
}

File: python/1101.py
# -*- coding: utf-8 -*-

n = 1
m = 1
soma = 0
while m >=1 or n >=1:
    a = [int(x) for x in input().split()]
    a.sort()
    m = a[0]
    n = a[1]
    if m <=0 or n<=0:
        break
    for x in range(m, n+1):
        print(x, end=' ')
        soma += x
    print('Sum=%r'%soma)
    soma = 0
